pkg/pillar/cmd/domainmgr/domainmgr.go

```diff
@@ -1332,7 +1332,13 @@ func configToStatus(ctx *domainContext, config types.DomainConfig,
 		ds.ReadOnly = dc.ReadOnly
 		ds.FileLocation = dc.FileLocation
 		ds.Format = dc.Format
-		ds.Devtype = dc.Devtype
+		// Generate Devtype for hypervisor package
```

Comment:
Don't know how this slipped past me @eriknordmark -- especially since I'm now running healdong into this same issue with my Task refactoring of the domainmgr. This whole Devtype vs. Vdev is a holdover mess from when we used to be Xen only btw and needs to be changes ASAP.

At any rate -- I filed a tracker story for both of us to tackle this. I'll do some as part of my work, but I'll need your help hooking into the guts of the new Volume implementation.